README.md

```diff
@@ -1,16 +1,23 @@
-# UEFI for PINE64 Quartz64 (Rockchip RK356X)
+# UEFI for Rockchip RK356X based SBR
 
-This repository contains a port of Tianocore EDK II to the PINE64 Quartz64 board.
+This repository contains a port of Tianocore EDK II to various Rockchip RK356x based boards:
 
-https://www.pine64.org/quartz64a/
+* [PINE64 Quartz64 A](https://www.pine64.org/quartz64a/) / [PINE64 Quartz64 B](https://www.pine64.org/quartz64b/) / [PINE64 SOQuartz](https://wiki.pine64.org/wiki/SOQuartz)
+* [Firefly roc-rk3566-pc](https://en.t-firefly.com/product/industry/rocrk3566pc.html) / [StationPC Station M2](https://www.stationpc.com/product/stationm2)
+* [Firefly roc-rk3568-pc](https://en.t-firefly.com/product/industry/rocrk3568pc.html) / [StationPC Station P2](https://www.stationpc.com/product/stationp2)
```

Comment:
For ROC-RK3568-PC, Station P2, and ROCK3 Compute Module, lets be clear that these are only available in source form at this time as the ports are works in progress.

Reply:
Valuable input: haven't noticed that.